tests/local/network.go

```diff
@@ -42,15 +44,13 @@ var _ interfaces.LocalNetwork = &LocalNetwork{}
 type LocalNetwork struct {
 	teleporterContractAddress common.Address
 	primaryNetworkInfo        *interfaces.SubnetTestInfo
-	subnetAID, subnetBID      ids.ID
 	subnetsInfo               map[ids.ID]*interfaces.SubnetTestInfo
-	subnetNodeNames           map[ids.ID][]string
+	extraNodes                []*tmpnet.Node
```

Comment:
Could you add a comment here explaining what `extraNodes` is intended to be used for, and the requirements for its use?

Reply:
addressed in https://github.com/ava-labs/teleporter/pull/401/commits/1e4e260794b27ac10ebb0e0d69aca70696290d12